vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
Yes, looks like the issue, although the test did not fail on that with the gccgo build so it is a bit odd. Maybe best to remove the pkgconfig part?